scripts/fbi/crime/preprocess.py

```diff
@@ -77,9 +77,9 @@
 Node: E:FBI_Crime->E{index}
 typeOf: dcs:StatVarObservation
 variableMeasured: dcs:{stat_var}
-measurementMethod: dcs:FBI_Crime
 observationAbout: C:FBI_Crime->GeoId
 observationDate: C:FBI_Crime->Year
+observationPeriod: "P1Y"
```

Comment:
For the record, the current MCF template is documented here: https://g3doc.corp.google.com/datacommons/mcf/crime/README.md?cl=355673889